exporter/doc/measure_section.go

```diff
@@ -0,0 +1,121 @@
+package doc
+
+import (
+	"bytes"
+	"fmt"
+	"text/template"
+
+	"github.com/projectcypress/cdatools/models"
+)
+
+type MeasureSection struct {
+	models.Measure
+	Results         map[string]AggregateCount
+	PerformanceRate PopulationGroup
+	Data            MeasureData
+}
+
+type AggregateCount struct {
+	Measure_HQMFID   string
+	Populations      []Population
+	PopulationGroups []PopulationGroup
+}
+
+func (ac AggregateCount) IsCV() bool {
+	for _, pop := range ac.Populations {
+		if pop.Type == "MSRPOPL" {
+			return true
+		}
+	}
+	return false
+}
+
+func (ac AggregateCount) FindObserv() *Population {
+	for _, pop := range ac.Populations {
+		if pop.Type == "OBSERV" {
+			return &pop
+		}
+	}
+	return nil
+}
+
+func (ms MeasureSection) CreateMeasureData(pop Population, obs Population) MeasureData {
```

Comment:
It doesn't seem like this function should be on `MeasureSection`, since it purely deals with `MeasureData` and doesn't reference `ms`. Should this be on the `MeasureData` interface? And also going off what @mattrbianchi said, it ought to be called `NewMeasureData`.